docs/troubleshooting.md

```diff
@@ -142,3 +142,11 @@ Based on the above log, you can add the following entry to remove the operation:
 ```json
 {"op":"remove", "path":"/spec/template/spec/hostNetwork"}
 ```
+
+### My bundle which has a Horizontal Pod Autoscaler is in state `Modified`
+
+For bundles with an HPA is expected that it is in state `Modified`, as the bundle contains fields that differ from the state of the Bundle at deployment.
+
+In such cases you have to define a patch in the `fleet.yaml` to ignore this field according to [bundle-diffs](./bundle-diffs.md).
+
+An example of configuring this for the replicas field of a Deployment can be found [here](https://github.com/axeal/fleet-test/blob/320229/fleet.yaml).
```

Comment:
```suggestion
Please refer [here](https://github.com/axeal/fleet-test/blob/320229/fleet.yaml) for an example of how to configure settings for the **replicas** field of a Deployment.
```

Reply:
Thanks for the review and the corrections. Please check my latest changes.